frontend/i18n-scripts/i18n-to-po.js

```diff
@@ -0,0 +1,146 @@
+const fs = require('fs');
+const path = require('path');
+const minimist = require('minimist');
+const { i18nextToPo } = require('i18next-conv');
+const common = require('./common.js');
+
+function save(target) {
+  return (result) => {
+    fs.writeFileSync(target, result);
+  };
+}
+
+function removeValues(i18nFile, filePath) {
+  const file = require(i18nFile);
+
+  const updatedFile = {};
+
+  const keys = Object.keys(file);
+
+  for (let i = 0; i < keys.length; i++) {
+    updatedFile[keys[i]] = '';
+  }
+
+  const tmpFile = fs.openSync(filePath, 'w');
+
+  fs.writeFileSync(tmpFile, JSON.stringify(updatedFile, null, 2));
+}
+
+function consolidateWithExistingTranslations(filePath, fileName, language, package) {
+  let existingTranslationsPath;
+  const englishFile = require(filePath);
+  const englishKeys = Object.keys(englishFile);
+  if (package) {
+    existingTranslationsPath = `./../packages/${package}/locales/${language}/${fileName}.json`;
+  } else {
+    existingTranslationsPath = `./../public/locales/${language}/${fileName}.json`;
+  }
+  if (fs.existsSync(path.join(__dirname, existingTranslationsPath))) {
+    const existingTranslationsFile = require(path.join(__dirname, existingTranslationsPath));
+    const existingKeys = Object.keys(existingTranslationsFile);
+    const matchingKeys = englishKeys.filter((k) => existingKeys.indexOf(k) > -1);
+
+    for (let i = 0; i < matchingKeys.length; i++) {
+      englishFile[matchingKeys[i]] = existingTranslationsFile[matchingKeys[i]];
+    }
+
+    const untrackedKeys = existingKeys.filter((k) => englishKeys.indexOf(k) === -1);
+    for (let i = 0; i < untrackedKeys.length; i++) {
+      englishFile[untrackedKeys[i]] = existingTranslationsFile[untrackedKeys[i]];
+    }
+
+    fs.writeFileSync(filePath, JSON.stringify(englishFile, null, 2));
+  }
+}
+
+function processFile(fileName, package, language) {
+  let tmpFile;
+  if (package) {
+    const i18nFile = path.join(__dirname, `./../packages/${package}/locales/en/${fileName}.json`);
+
+    fs.mkdirSync(path.join(__dirname, `./../packages/${package}/locales/tmp`), {
+      recursive: true,
+    });
+
+    tmpFile = path.join(__dirname, `./../packages/${package}/locales/tmp/${fileName}.json`);
+
+    removeValues(i18nFile, tmpFile);
+    consolidateWithExistingTranslations(tmpFile, fileName, language, package);
+
+    fs.mkdirSync(path.join(__dirname, `./../public/locales/pos/${language}`), {
```

Comment:
The generated files should go into a different folder that has a gitignore

Reply:
A sibling folder to `frontend/public`. `public` is everything that we ship as part of the app itself.

Reply:
Thanks. Do you care what the new folder is named?

Reply:
@spadgett - I changed it to `frontend/po-files`; let me know if you want a different name. I added a `.gitignore`.